src/component/PriceCard.jsx

```diff
@@ -0,0 +1,56 @@
+import "../styles/pricecard.css";
+
+function PriceCard() {
+  return (
+    <div className="price-conteiner">
+      <h2>Select your plan</h2>
+      <div className="price-row">
+        <div className="price-col">
+          <p>Basic</p>
+          <h3> 199dkk <span>/ month</span></h3>
```

Comment:
Will this information come from the database or will always be set and hard-coded? 

Either way, such implementation is a bit too raw and difficult to maintain. 

Try to establish a data object and loop through it to render the list..

It would also be beneficial to have some different components (they can be here in the same file in this case) to render the different repetitive parts.
Refresh on React Thinking https://react.dev/learn/thinking-in-react and try to make this a bit more React-y 🙏🏽

Reply:
Created new pricing card using data object and render.